### PR TITLE
Rejected errors fix links

### PR DESCRIPTION
**Pull request by stuartmoore1023**
PTAL

### CODE REVIEW

File: jekyll/_docs/airbrake-faq/rejected-errors.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ The error notice is rejected if it is sent with incorrect JSON/XML as determined
 by:
 
 - **JSON:** [POST data schema](https://airbrake.io/docs/api/#post-data-schema-v3) and
-[POST data fields](https:/airbrake.io/docs/api/#post-data-fields-v3)
-- **XML**: as described in our [XML API doc](/docs/api-2/notifier-api-v23)
+[POST data fields](https://airbrake.io/docs/api/#post-data-fields-v3)
+- **XML**: as described in our [XML API doc](https://airbrake.io/docs/legacy-xml-api/)
```
Comment (thompiler) [inline jekyll/_docs/airbrake-faq/rejected-errors.md]:
```suggestion
- **XML**: as described in our [XML API doc](/docs/legacy-xml-api/)
```

We shouldn't hardcode the host (`https://airbrake.io`) for any page within the docs. Using just `/docs/legacy-xml-api/` links to the correct doc in whatever environment the doc is being run in like staging or development (which each have different hosts).

**Comment by stuartmoore1023** [comment]
@thompiler I tried that and I get a 404. Just double checked. Am I doing something wrong?

**Comment by thompiler** [comment]
>@thompiler I tried that and I get a 404. Just double checked. Am I doing something wrong?

Oh hmm I'm sure I tested it. What exactly did you do to get the 404?

**Comment by stuartmoore1023** [comment]
I left the change in place. You can test it.

**Comment by thompiler** [comment]
@stuartmoore1023 I am not seeing a 404 in development. Can you check again?

Also, if you want to, you can incorporate a suggested change (like I made) into your PR as a commit by clicking this button:

Screen Shot 2019-12-12 at 1 42 16 PM

That's only if you actually want to make that change. It's up to you since it's your PR.

**Comment by thompiler** [comment]
@stuartmoore1023 I am not seeing anything odd. Can you please check again?

**Pull request merged**